### DISCUSSION TITLE
Is there a way to tell Renovate to ignore a specific instance of a dependency?

### DISCUSSION THREAD

**Discussion by deviantintegral** [renovatebot/renovate#22760] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

For PHP libraries, we often want to test against multiple versions of PHP. Here's an example where we use different base images for PHP 8.0, 8.1, and 8.2:

https://github.com/deviantintegral/composer-gavel/blob/f5f9d0c241eb361c40471206bd03e93a286314ed/.circleci/config.yml#L113-L142

Renovate will attempt to upgrade the older versions to 8.2: https://github.com/deviantintegral/composer-gavel/pull/45

While we can close the PR, in practice upgrading those jobs is a manual process whenever the next PHP release comes out.

Is there a way to mark a specific line in a file with a comment to tell Renovate to ignore it? 

### What you tried so far.

I considered using [allowedVersions](https://docs.renovatebot.com/configuration-options/#allowedversions) but it looks like there's no way to specify an instance of a version such as "never upgrade PHP 8.1 but allow PHP 8.2".

**Comment by rarkins** [comment, marked as answer]
It looks like you don't want to update those values at all? If so then you can match on file name, package name, and current version, and then set enabled=false

**Reply by deviantintegral** [reply]
Thank you! This worked perfectly.

https://github.com/deviantintegral/har/blob/7536d783c52d6f8e2d96273bb4e5497904bd9c48/renovate.json#L13-L22
